app/src/main/res/values-fr/strings.xml

```diff
@@ -11,8 +11,320 @@
+  </string>
+  <string name="troubleshoot_package_server_misconfigured">Paramètre de serveur de package peut être incorrect</string>
+  <string name="troubleshoot_update_server_misconfigured_details">
+      S\'il vous plaît vérifier que les paramètres du serveur de package est correct ou consultez un administrateur de l\'aide.
+  </string>
+
+  <string name="present_filter_description">Tous les présents patients</string>
+  <string name="age_filter_description">Sous %d années</string>
+  <string name="filter_group_default_name">Plusieurs filtres</string>
+
+  <string name="patient_missing_age">Entrez l\'âge (en utilisant années, mois, ou les deux)</string>
+  <string name="patient_validation_missing_family_name">Entrez le nom de famille</string>
+  <string name="patient_validation_missing_given_name">Entrez le nom donné</string>
+  <string name="patient_validation_missing_id">Entrez l\'ID du patient</string>
+  <string name="patient_validation_missing_admission_date">Entrez la date d\'admission</string>
+  <string name="patient_missing_gender">S\'il vous plaît sélectionnez Masculin ou Féminin.</string>
+  <string name="patient_validation_future_admission_date">Date d\'admission ne peut pas être dans l\'avenir.</string>
+  <string name="patient_validation_future_onset_date">Symptôme date de début ne peut pas être dans l\'avenir.</string>
+  <string name="patient_creation_clear_symptom_onset_date">Clair</string>
+  <!-- Use en dashes for unknown names. These names will be searchable using any form of dash. -->
+  <string name="family_name_hint_unknown">–</string>
+  <string name="given_name_hint_unknown">–</string>
+  <string name="unknown_name">–</string>
+
+  <!-- UI strings related to settings (see default_settings.xml for default values) -->
+  <string name="title_activity_settings">Paramètres</string>
+
+  <!-- Settings pages -->
+  <string name="pref_header_general">Général</string>
+  <string name="pref_header_advanced">Avancée</string>
+  <string name="pref_header_developer">Développeur</string>
+
+  <!-- Preference titles and descriptions -->
+  <string name="pref_title_server">Buendia server</string>
+  <string name="pref_title_openmrs_user">OpenMRS nom d\'utilisateur</string>
+  <string name="pref_title_openmrs_password">OpenMRS mot de passe</string>
+  <string name="pref_title_openmrs_root_url">OpenMRS URL de base</string>
+  <string name="pref_title_package_server_url">Package URL du serveur</string>
+
+  <string name="pref_title_apk_update_interval_secs">APK mise à jour Intervalle de vérification (secondes)</string>
+  <string name="pref_title_store_form_instances">Instances de formulaire de stocker localement</string>
+  <string name="pref_desc_store_form_instances">Normalement formes seront supprimés après avoir été envoyé au serveur. Sélectionnez cette option pour les garder pour le débogage.</string>
+  <string name="pref_title_xform_update_client_cache">Mettre à jour observations locales sur la soumission</string>
```

Comment:
Add 'des' into 'Mettre à jour des observations ...'
I'm not sure if the preposition 'sur' is the correct form  in  'locales sur la soumission'